src/sidebar/oauth-auth.js

```diff
@@ -86,6 +99,42 @@ function auth($http, $window, flash, random, settings) {
     return $http.post(tokenUrl, data, requestConfig);
   }
 
+  function grantTokenFromHostPage() {
+    var cfg = serviceConfig(settings);
+    if (!cfg) {
+      return null;
+    }
+    return cfg.grantToken;
+  }
+
+  /**
+   * Read the last-saved access/refresh tokens for `authority`.
+   */
+  function readLastUsedToken(authority) {
```

Comment:
Hmm ... how about I change this to `loadToken` for consistency with `saveToken`?